db/migrate/20210714134353_add_import_to_finding_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddImportToFindingAnswers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :finding_answers, :imported, :boolean, default: false
```

Comment:
```suggestion
    add_column :finding_answers, :imported, :boolean, default: false, null: false
```
Solo para que no quede un "ternario".

Reply:
impecable